searchlib/src/vespa/searchlib/tensor/hnsw_index.cpp

```diff
@@ -310,6 +310,43 @@ HnswIndex::remove_document(uint32_t docid)
     _node_refs[docid].store_release(invalid);
 }
 
+std::vector<uint32_t>
+HnswIndex::find_top_k(TypedCells vector, uint32_t k)
+{
+    std::vector<uint32_t> result;
+    std::vector<HnswCandidate> candidates = top_k_candidates(vector, k + 100);
```

Comment:
Perhaps 'k + 100' should be a parameter to the function instead, e.g. 'search_k'?

Reply:
now exposed as "explore_k"